user-service/src/main/java/com/movie/recommender/user/config/SecurityConfig.java

```diff
@@ -21,7 +21,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .sessionManagement(session -> session
                         .sessionCreationPolicy(SessionCreationPolicy.STATELESS))
                 .authorizeHttpRequests(authorize -> authorize
-                        .requestMatchers("/api/v1/users/register", "/api/v1/users/login").permitAll()
+                        .requestMatchers("/api/v1/users/register", "/api/v1/users/login", "/api/v1/users/{userId}/favorites").permitAll()
```

Comment:
using path variables like `{userId}` in `requestMatchers` is not supported
I recommend using the `/api/v1/favorites` path for the `FavoritesController` and making `userId` part of the method mappings
for example` @GetMapping("/{userId}")`